netket/graph/symmetry.py

```diff
@@ -101,18 +105,21 @@ def inverse(self):
 
         return inverse
 
-    def product_table(self):
+    def __product_table(self, inverse):
```

Comment:
Does `inverse` really have to be passed explicitly here? The caching also works if, as before, `self.inverse()` is called in the body of this function. Or am I missing something?

Reply:
It definitely does not. I can change it back